core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -351,10 +351,11 @@ public void setAcl(Supplier<JournalContext> context, SetAclEntry entry) {
    */
   @Nullable
   public String getRootUserName() {
-    if (mState.getRoot() == null) {
+    InodeDirectory root = mState.getRoot();
+    if (root == null) {
       return null;
     }
-    return mState.getRoot().getOwner();
+    return root.getOwner();
```

Comment:
I know this is old code, but I don't think returning null here makes any sense. If you throw an error if root, is null, which will break?
1. Start alluxio from no journal
2. Start alluxio from existing journal
3. Run all UTs

Reply:
@jiacheliu3 After go through the old code, I agree it is impossible to return null while invoke `getRoot` for now,
since the root should be a directory all the time.

```
  public InodeDirectory getRoot() {
    return mInodeStore.get(0).map(Inode::asDirectory).orElse(null);
  }
```